internal/cmd/cmd.go

```diff
@@ -1,4 +1,6 @@
-package hcsoci
+// Package cmd provides functionality used to execute commands inside of containers
+// or UVMs, and to connect an upstream client to those commands for handling in/out/err IO.
```

Comment:
Can we just say for handling Stdio?

Reply:
This was what Kevin had suggested. I don't see a need to change it unless it's unclear.

Reply:
Its not unclear, I don't think anyone would not understand. Was mostly a nit

Reply:
If you agree the other wording is clearer, I'm happy with it.

Reply:
I think that both wordings are clear, though the current one is more explicit. I don't know if I care either way haha